package.json

```diff
@@ -47,20 +47,25 @@
   "license": "MIT",
   "dependencies": {
     "cheerio": "^0.19.0",
+    "object-assign": "^4.0.1",
```

Comment:
please use `object.assign` instead of `object-assign` - it's more spec-compliant.